conda-recipe/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: csxtools
+  version: {{ environ.get('GIT_DESCRIBE_TAG', 'GIT_STUB') }}.post{{ environ.get('GIT_DESCRIBE_NUMBER','') }}
```

Comment:
There are no tags yet. @stuwilkins what is your planned tagging scheme?  Basically 'v0.0.0' or '0.0.0'?